about.markdown

```diff
@@ -1,18 +1,73 @@
 ---
 layout: page
-title: About
+title: What is code curious
 permalink: /about/
 ---
 
-This is the base Jekyll theme. You can find out more info about customizing your Jekyll theme, as well as basic Jekyll usage documentation at [jekyllrb.com](https://jekyllrb.com/)
+## What is our mission?
 
-You can find the source code for Minima at GitHub:
-[jekyll][jekyll-organization] /
-[minima](https://github.com/jekyll/minima)
+<!-- Founded in April 2012, under our former name Rails Girls Berlin is the daughter of the **worldwide non-profit organization** [Rails Girls](http://railsgirls.com) that started in Finland. -->
```

Comment:
There's something wrong with this sentence grammatically I think. Either it is 

`Founded in April 2012 and formerly know as [...] is the daughter`
or
`[...] we are a daughter/child organization of the [...]`

Reply:
Yes, it's a comment though. I'll delete it to decrease confusion :)